docs/pages/setup/admin/daemon.mdx

```diff
@@ -45,17 +67,91 @@ PIDFile=/run/teleport.pid
 WantedBy=multi-user.target
 ```
 
-## Daemon restarts
+<Notice type="tip">
+
+If `/usr/lib/systemd/system/` does not exist, consult the list of
+[unit file load paths](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Unit%20File%20Load%20Path)
+for other supported paths.
+
+</Notice>
+
+Enable the unit so systemd can place it in its dependency tree:
+
+```code
+$ sudo systemctl enable teleport.service
+Created symlink /etc/systemd/system/multi-user.target.wants/teleport.service → /lib/systemd/system/teleport.service.
+```
+
+Start the unit:
 
-As covered in the [Graceful Restarts](./graceful-restarts.mdx) section, Teleport
-supports graceful restarts. To upgrade a host to a newer Teleport version, an
-administrator must:
+```code
+$ sudo systemctl start teleport.service
```

Comment:
Nit: Here and below you don't actually need to specify the `.service` suffix, should just be able to `systemctl start/status teleport`.